src/gnatss/constants/__init__.py

```diff
@@ -108,3 +108,75 @@
 ANTENNA_NORTHWARD = "ant_n"
 ANTENNA_UPWARD = "ant_u"
 ANTENNA_DIRECTIONS = [ANTENNA_EASTWARD, ANTENNA_NORTHWARD, ANTENNA_UPWARD]
+
+L1_DATA_FORMAT = {
+    "INSPVAA": {
+        "regex_pattern": r"#INSPVAA,.*?,.*?,.*?,.*?,(.*?),(.*?),"
+        r".*?,.*?,.*?;.*?,.*?,(.*?),"
+        r"(.*?),(.*?),(.*?),(.*?),(.*?),(.*?),"
+        r"(.*?),(.*?),(.*?)\*+.*\n",
+        "data_fields": (
+            "Week",
+            "Seconds",
+            "Latitude",
+            "Longitude #",
+            "Height",
+            "North Velocity",
+            "East Velocity",
+            "Up Velocity",
+            RPH_ROLL,
+            RPH_PITCH,
+            RPH_HEADING,
+            "Status",
+        ),
+        "data_fields_dtypes": (
+            "int",
+            "float",
+            "float",
+            "float",
+            "float",
+            "float",
+            "float",
+            "float",
+            "float",
+            "float",
+            "float",
+            "object",
+        ),
+    },
+    "INSSTDEVA": {
+        "regex_pattern": r"#INSSTDEVA,.*?,.*?,.*?,.*?,(.*?),(.*?),.*?,.*?,.*?;(.*?),(.*?),"
```

Comment:
This is so clever! Looking good on parsing week and seconds from header with regex 👍🏽